custom/icds_reports/utils/__init__.py

```diff
@@ -1592,3 +1592,11 @@ def get_datatables_ordering_info(request):
 
 def phone_number_function(x):
     return "+{0}{1}".format('' if str(x).startswith('91') else '91', x) if x else x
+
```

Comment:
I have already merged my fix with master which has this function. can you merge you branch with latest master and resolve the merge conflicts. which will provide you the latest code